pkg/controller/podautoscaler/horizontal.go

```diff
@@ -435,17 +435,26 @@ func (a *HorizontalController) reconcileAutoscaler(hpav1Shared *autoscalingv1.Ho
 		// bogus CPU usage report from heapster/kubelet (like in issue #32304).
 		scaleUpLimit := calculateScaleUpLimit(currentReplicas)
 
+		replicaCountZeroStatusMsg := "the desired replica count was zero"
```

Comment:
I'd either a) make this a constant, or b) just inline it, since it's only used in two spots, and they're right next to each other.

Reply:
Sounds good - think I'll go within inlining because - as you say, its only used in two spots and they're right next to each other.

Reply:
@DirectXMan12 - done, whenever you get a chance for another look. Thanks :)

Out of curiosity, what is the preference in k8s - amending the original commit when its only small changes, or creating a new commit with the small change and then squashing at the end?